java/test/apps/tests/AllTest.java

```diff
@@ -41,8 +41,10 @@ public static Test suite() {
         // all tests from here down in heirarchy
         TestSuite suite = new TestSuite("AllTest");  // no tests in this class itself
         // all tests from other classes
-        suite.addTest(jmri.PackageTest.suite());
-        suite.addTest(new junit.framework.JUnit4TestAdapter(apps.PackageTest.class));
+        suite.addTest(jmri.jmrit.logix.OBlockTest.suite());
+//        suite.addTest(jmri.jmrit.logix.PackageTest.suite());
+//        suite.addTest(jmri.PackageTest.suite());
+//        suite.addTest(new junit.framework.JUnit4TestAdapter(apps.PackageTest.class));
```

Comment:
Commenting out the jmri.PackageTest and apps.PackageTest suites is what caused the two travis tests to fail, due to the drop in test coverage.

My guess is that this file was committed in error.